### ISSUE TITLE
No error message on connection error

### ISSUE THREAD

**Issue by txau** [Wizcorp/node-graylog2#35]
It took us a while to figure out why our messages were not being logged. 

Our GrayLog2 GELF port was firewalled, but we didn't receive any feedback at all. It would be great to have more reliable error messages.

Thanks!

**Comment by stelcheck** [comment]
I don't believe GELF can provide any sort of connection error feedback; GELF is UDP based, so there are no connections in the first place - only messages being passed from one port to another.
